theme/errorList.less

```diff
@@ -17,8 +17,12 @@
     }
 
     .errorListInner {
-        height: 200px;
+        height: 200rem;
```

Comment:
200rem === 3200px (1 rem == fontsize of root element); probably not quite what you want :) (agreed that it looks great otherwise though!)